msi/GitProduct.wxs

```diff
@@ -70,6 +70,9 @@
         <Environment Name='PATH' Value='[BinFolder]' Action='set' Part='last' System='yes' />
         <RemoveFolder Directory='GitProgramMenuFolder' On='uninstall' />
       </Component>
+      <Component Directory="INSTALLFOLDER">
+        <RegistryValue Root="HKLM" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="$(var.Version)" />
+      </Component>
```

Comment:
Hmm, must've gotten put in a different commit during a fixup. I'll correct this situation--that change should be here.